net-misc/gns3-server/gns3-server-2.1.20.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit distutils-r1 python-utils-r1
+
+DESCRIPTION="GNS3 server to asynchronously manage emulators"
+HOMEPAGE="http://www.gns3.net/"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-python/jsonschema-2.6.0[${PYTHON_USEDEP}]
+	<dev-python/aiohttp-2.4.0[${PYTHON_USEDEP}]
+	>=dev-python/aiohttp-2.3.3[${PYTHON_USEDEP}]
+	>=dev-python/aiohttp-cors-0.6.0[${PYTHON_USEDEP}]
+	>=dev-python/jinja-2.7.3[${PYTHON_USEDEP}]
+	>=dev-python/raven-5.23.0[${PYTHON_USEDEP}]
+	>=dev-python/psutil-3.0.0[${PYTHON_USEDEP}]
+	>=dev-python/python-zipstream-1.1.4[${PYTHON_USEDEP}]
+	dev-python/prompt_toolkit[${PYTHON_USEDEP}]
+	<dev-python/async_timeout-3.0.0[${PYTHON_USEDEP}]
+	>=net-misc/ubridge-0.9.14
+	>=app-emulation/dynamips-0.2.18
+	sys-apps/busybox
+"
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+src_prepare() {
+	default
+
+	#Remove Pre-built busybox binary
+	rm gns3server/compute/docker/resources/bin/busybox || die
+	# Package installs 'tests' package which is forbidden
+	rm -rf tests || die
+}
+
+python_install() {
+	distutils-r1_python_install
+
+	dodir "$(python_get_sitedir)/gns3server/compute/docker/resources/bin"
```

Comment:
This is wrong, python_get_sitedir is EPREFIX-ed, so `do*` functions add a second prefix inside.

Reply:
this installs fine, where 'mkdir' or 'ln -s' doesn't
do you recommend doing this some other way?

Reply:
`mkdir -p "${ROOT}...` should work fine.

Reply:
sorry so I tried that and it doesn't work, it really don't look to me like python_get_sitedir contains eprefix
```
Installing gns3vmnet script to /var/tmp/portage/net-misc/gns3-server-2.1.20/image/_python3.6/usr/lib/python-exec/python3.6
mkdir: cannot create directory ‘/usr/lib64/python3.6/site-packages/gns3server/compute/docker/resources/bin’: Permission denied
ln: failed to create symbolic link '/usr/lib64/python3.6/site-packages/gns3server/compute/docker/resources/bin/busybox': No such file or directory
 * python3_6: running distutils-r1_run_phase distutils-r1_python_install_all
```

Reply:
Sorry, s/ROOT/D/